test/unit/BPool.t.sol

```diff
@@ -109,6 +113,28 @@ abstract contract BasePoolTest is Test, BConst, Utils, BMath {
     uint256 _foo = bpow(_y, _weightRatio);
     vm.assume(bsub(_foo, BONE) < type(uint256).max / _tokenInBalance);
   }
+
+  function _assumeCalcPoolOutGivenIn(
+    uint256 _tokenInDenorm,
+    uint256 _tokenInBalance,
+    uint256 _tokenAmountIn,
+    uint256 _swapFee,
+    uint256 _totalWeight,
+    uint256 _totalSupply
+  ) internal pure {
+    uint256 _normalizedWeight = bdiv(_tokenInDenorm, _totalWeight);
+    vm.assume(_normalizedWeight < BONE); // TODO: why this? if the weights are between allowed it should be fine
```

Comment:
isn't it better to bound the tokenInDenorm between `MIN_WEIGHT` and `MAX_TOTAL_WEIGHT`?

Reply:
actually `MAX_WEIGHT` and `MAX_TOTAL_WEIGHT` are the same, I should change the bounds to adapt to this. That will likely fix the problem